pkg/probe/fortiswitch_health.go

```diff
@@ -0,0 +1,151 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func probeSwitchHealth(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		mSumCPU = prometheus.NewDesc(
+			"fortiswitch_health_summary_cpu",
+			"Summary CPU health",
+			[]string{"rating", "fortiswitch", "VDOM"}, nil,
+		)
+		mSumMem = prometheus.NewDesc(
+			"fortiswitch_health_summary_mem",
+			"Summary MEM health",
+			[]string{"rating", "fortiswitch", "VDOM"}, nil,
+		)
+		mSumUpTime = prometheus.NewDesc(
+			"fortiswitch_health_summary_uptime",
+			"Summary Uptime",
```

Comment:
If you have the documentation it would be really great if these descriptions could be made more verbose so I for example have any idea what these means. But if you don't, it's fine - sometimes we simply don't know